js/views/settings.js

```diff
@@ -176,7 +180,21 @@ app.views.Settings = (function() {
 				errors.push(app.i18n.t('settings.at-least-one-crypto-currency-required'));
 			}
 
-			return errors;
+
+			_.each(data.acceptCryptoCurrencies, _.bind(function(currency) {
+
+				app.paymentMethods[currency].validateAsync(data, function(error, validatedData) {
+
+					if (error) {
+						cb(error);
```

Comment:
Callback function here will be executed multiple times if more than one payment method has a validateAsync function.